analytics-sidecar-client-common/src/test/java/org/apache/cassandra/sidecar/common/utils/TimeUtilsTest.java

```diff
@@ -55,8 +54,8 @@ public void testRandomDurationWithMinimumBelowMaximum()
             Duration maximum = Duration.ofSeconds(test + 1);
             Duration random = TimeUtils.randomDuration(minimum, maximum);
 
-            assertTrue(minimum.compareTo(random) <= 0);
-            assertTrue(random.compareTo(maximum) <= 0);
+            assertThat(minimum.compareTo(random) <= 0).isTrue();
+            assertThat(random.compareTo(maximum) <= 0).isTrue();
```

Comment:
"You" missed this one
```suggestion
            assertThat(minimum.compareTo(random)).isLessThanOrEqualTo(0);
            assertThat(random.compareTo(maximum)).isLessThanOrEqualTo(0);
```